src/viur_cli/cloud.py

```diff
@@ -352,7 +352,9 @@ def gcloud_setup():
 # Helper function for running Commands in subprocess and getting the Output
 def run_command(command):
     try:
-        subprocess.run(command, check=True, shell=True, capture_output=True)
+        output = subprocess.check_output(command, shell=True)
+        print(output)
+        return output
```

Comment:
Looks like a forgotten debug print. Or not?

```suggestion
        return subprocess.check_output(command, shell=True)
```